nat/nat.go

```diff
@@ -34,6 +34,9 @@ func NewPort(proto, port string) Port {
 }
 
 func ParsePort(rawPort string) (int, error) {
+	if len(rawPort) == 0 {
+		return 0, nil
```

Comment:
@thaJeztah Well I am not sure, I needed it to fix an error case and it makes sense to me, but maybe I could be missing something. That's why I need people to review it. @LK4D4 maybe you know why this would be bad?